client/packages/invoices/src/Returns/modals/InboundReturn/InboundReturn.tsx

```diff
@@ -117,7 +127,7 @@ export const InboundReturnEditModal = ({
     <DialogButton
       onClick={handleNextItem}
       variant="next"
-      disabled={currentTab !== Tabs.Reason}
+      disabled={currentTab !== Tabs.Reason || (isDisabled && !hasNextItem)}
```

Comment:
The "Okay and Next" button should be disabled when `isDisabled` and we've got to the end of the list, as we don't want to be able to make a new line. (this is why I needed to explicitly pass `hasNextItem` in)

Reply:
For consistency... the other modals disable Ok & Next when you get to the end of the list, regardless of whether the invoice is disabled. If you want to add new lines, you need to click the add but explicitly. Could we make this the same?

Reply:
Oh, I didn't realise that. I guess that makes sense, maybe, I can see it both ways. But sure, will keep it consistent.

Reply:
Done in 5c125e06c